From 6f00d7346c35f46801c71cdecbba843336a53ad3 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Mon, 5 Nov 2018 15:04:05 +0900
Subject: [PATCH 336/909] drm: rcar-du: Add depends on DRM_RCAR_DU to config
 DRM_RCAR_LVDS

commit 1cfe50a84da3526fd40bb8c07fba7fbb71ed5370 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

With the LVDS encoder alone, it can not be displayed, it seems that
the requirement that DRM_RCAR_DU config must be Y when DRM_RCAR_LVDS
config is Y. So since LVDS encoder must depend on DRM_RCAR_DU,
it is added with this patch.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/rcar-du/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rcar-du/Kconfig b/drivers/gpu/drm/rcar-du/Kconfig
index 266ec2510f73..aab74309f941 100644
--- a/drivers/gpu/drm/rcar-du/Kconfig
+++ b/drivers/gpu/drm/rcar-du/Kconfig
@@ -21,6 +21,7 @@ config DRM_RCAR_DW_HDMI
 config DRM_RCAR_LVDS
 	tristate "R-Car DU LVDS Encoder Support"
 	depends on DRM && DRM_BRIDGE && OF
+	depends on DRM_RCAR_DU
 	select DRM_PANEL
 	help
 	  Enable support for the R-Car Display Unit embedded LVDS encoders.
-- 
2.17.1

